packages/deployments/contracts/src/init/helpers/tx.ts

```diff
@@ -6,15 +6,18 @@ import { CallSchema, Deployment } from "./types";
 
 const DEFAULT_CONFIRMATIONS = 3;
 
-export const waitForTx = async (args: {
+type WaitForTxArguments = {
   deployment: Deployment;
   tx: providers.TransactionResponse;
   name: string;
   checkResult?: {
     method: () => Promise<any>;
-    desired: any;
+    desired?: any;
```

Comment:
if you don't want to check the result, don't pass in `checkResult` method. I don't see a great reason to make `desired` optional...

Reply:
Lol I want to checkResult which isn't throwing, but don't know `desired` value.  i guess we can make it better later